azkaban/remote.py

```diff
@@ -546,14 +546,15 @@ def unschedule_workflow(self, name, flow):
     self._logger.info('Unscheduled project %s workflow %s.', name, flow)
     return res
 
-  def schedule_cron_workflow(self, name, flow, cron_expression, **kwargs):
+  def schedule_cron_workflow(self, name, flow, cron_expression, timezone, **kwargs):
```

Comment:
Can the `timezone` argument be made optional here too? What happens when we omit it (or send an empty string) to the Azkaban server?